great-idea-website/index.html

```diff
@@ -13,51 +13,78 @@
-  Innovation
-  On
-  Demand
-
-  Get Started
-
+    <div class="container">
+   <section>
+       <div class="main-header">
+        <header>
+            <nav>
+                <a href="">Services</a>
+                <a href="">Product</a>
+                <a href="">Vision</a>
+                <a href="">Features</a>
+                <a href="">About</a>
+                <a href="">Contact</a>
+            </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+             </header>
+           </div>
+    </section>
+       <section>
+        <div class="demand-main">
+           <div class="demand">
+           <h1>Innovation <br>
+           On<br>
+           Demand</h1>
+
+            <button>Get Started</button>
+           </div>
   <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
+           </div>
+   </section>
+<hr>
+   <section class="section">
+      <div class="features">
+        <h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+        <div class="about">
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+        </section>
+          <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
```

Comment:
Since the image is for mainly visual interest and not content with meaning, it might make more sense for the image to be out on its own instead of semantically grouped with these text sections. I feel styling might be easier too having the image and the top and bottom sections to be grouped separately.